crates/holochain/src/core/ribosome/host_fn/zome_info.rs

```diff
@@ -90,14 +112,33 @@ pub mod test {
                 FunctionName::new("zome_info"),
             ],
         );
-        let entries = vec![(ZomeIndex(0), vec![EntryDefIndex(0), EntryDefIndex(1)])];
-        let links = vec![(ZomeIndex(0), vec![])];
+    }
+
+
+    // Same test, but excluding wasmer metering extern fns
+    #[cfg(feature = "wasmer_wamr")]
+    #[tokio::test(flavor = "multi_thread")]
+    async fn zome_info_extern_fns_test() {
```

Comment:
- extracted extern_fns assertion from `zome_info_test` into its own
- made 2 feature-gated versions of the extern_fns_test for wasmer_sys and wasmer_wamr